draft-ietf-acme-acme.md

```diff
@@ -1276,12 +1277,11 @@ The server MUST include a Cache-Control header field with the "no-store"
 directive in responses for the new-nonce resource, in order to prevent
 caching of this resource.
 
-## Account Creation
+## Account Management 
 
 A client creates a new account with the server by sending a POST request to the
```

Comment:
Shouldn't it better be "... creates a new account, or manages an existing account, with the server by ..." (or something like that)?

Reply:
The start of management is creation :)

Reply:
It is a bit strange if you first read "A client creates a new account [...] by sending a POST request ...", and then you read in the details that the client can also send a parameter `onlyReturnExisting`. I don't mind too much if it is kept like this, though :)

Reply:
Added a bit of intro text to make this nicer.